frontend/storm_wiki/util/consts.py

```diff
@@ -0,0 +1,525 @@
+import os
+
+DB_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), "db", "settings.db")
+
+SEARCH_ENGINES = {
+    "searxng": {
+        "env_var": "SEARXNG_BASE_URL",
+        "settings": {
+            "base_url": {"type": "text", "required": True, "label": "SearXNG Base URL"},
+            "api_key": {
+                "type": "password",
+                "required": False,
+                "label": "SearXNG API Key (optional)",
+            },
+        },
+    },
+    "bing": {
+        "env_var": "BING_SEARCH_API_KEY",
+        "settings": {
+            "api_key": {"type": "password", "required": True, "label": "Bing API Key"},
+        },
+    },
+    "yourdm": {
```

Comment:
Maybe change to `you.com` to be clearer. I am not sure what `YourDM` means, guess it's a typo of `YouRM`.

Reply:
got it.